pipelines/perf-eval/Storage Benchmark/k8s-os-disk-fio.yml

```
@@ -0,0 +1,121 @@
+trigger: none
+schedules:
+  - cron: "0 10 */2 * *"
+    displayName: "Every Even Day at 10 AM"
+    branches:
+      include:
+        - main
+    always: true
+  - cron: "0 10 * * 1"
+    displayName: "Every Week on Monday at 10 AM"
+    branches:
+      include:
+        - main
+    always: true
+
+variables:
+  SCENARIO_TYPE: perf-eval
+  SCENARIO_NAME: k8s-os-disk
+
+stages:
+  - stage: azure_eastus2_managed
```

```suggestion
  - stage: azure_westus3_managed
```
Update stage name based on the region.